services/schedview/templates/deployment.yaml

```diff
@@ -0,0 +1,144 @@
+apiVersion: apps/v1
```

Comment:
You need to go through this file and remove the CSC configuration stuff since the application is not using DDS.